bittide/src/Bittide/ClockControl/Si539xSpi.hs

```diff
@@ -304,6 +306,72 @@ si539xSpiDriver SNat incomingOpS miso = (fromSlave, decoderBusy, spiOut)
 
 {-# NOINLINE si539xSpiDriver #-}
 
+-- | Consumes 'SpeedChange's produced by a clock control algorithm and produces a
+-- 'RegisterOperation' for the 'si539xSpi' core. Consumption rate of 'SpeedUp's and
```

Comment:
Could you add to the documentation that this is an alternative to controlling the FDEC/FINC pins directly? Also document its drawbacks/advantages.